physcraper/__init__.py

```diff
@@ -1298,47 +1268,16 @@ def __init__(self, config_obj, workdir, mrca=None):
         else:  # ncbi parser contains information about spn, tax_id, and ranks
             self.ncbi_parser = ncbi_data_parser.Parser(names_file=self.config.ncbi_parser_names_fn,
                                                        nodes_file=self.config.ncbi_parser_nodes_fn)
-        if self.mrca_ott is not None:
-            self.get_ncbi_mrca()
+        self.acc_tax_seq_dict = {} 
 
-    def get_ncbi_mrca(self):
-        """ get the ncbi tax ids from a list of mrca ott ids.
-        """
-        if type(self.mrca_ott) is not int:
-            for ott_id in self.mrca_ott:
-                ncbi_id = self.ottid_to_ncbiid(ott_id)
-                if ncbi_id is not None:
-                    self.mrca_ncbi.add(ncbi_id)
-        else:
-            ncbi_id = self.ottid_to_ncbiid(self.mrca_ott)
-            if ncbi_id is not None:
-                self.mrca_ncbi.add(ncbi_id)
+#moved self.get_ncbi_mrca to physcraper data object
 
-    def ottid_to_ncbiid(self, ott_id):
-        """ Find ncbi id for ott id.
-        Is only used for the mrca list thing!
-        """
-        debug("ottid to ncbiid")
-        debug(ott_id)
-        if ott_id in self.ott_to_ncbi:
-            ncbi_id = self.ott_to_ncbi[ott_id]
-        elif ott_id in self.ott_to_name:
-            ott_name = self.ott_to_name[ott_id]
-            if self.config.blast_loc == "remote":
-                ncbi_id = self.get_rank_info_from_web(ott_name)
-            else:
-                ncbi_id = self.ncbi_parser.get_id_from_name(ott_name)
-        else:  # with new ncbi taxa there might be no match in ott_to_ncbi
-            tx = APIWrapper().taxomachine
-            nms = tx.taxon(ott_id)
-            # debug(nms)
-            ott_name = nms[u"unique_name"]
-            ncbi_id = None
-            if u"ncbi" in nms[u"tax_sources"]:
-                ncbi_id = nms[u"tax_sources"][u"ncbi"]
-        return ncbi_id
 
-    def get_ncbiid_from_tax_name(self, tax_name):
+# removed funtion ottid_to_ncbiid.
+# if ott is isn't mapped to ncbi_id in the opentree taxonomy, (ott_to_ncbi) we shouldn't map it here.
+
+    def get_ncbiid_from_tax_name(self, tax_name): 
+    #TODO when do we need this? Is this just when we read in the original data?
```

Comment:
This is used by get_rank_info_from_web(), a  function only used for the physcraper light version and which we decided on to work for v2.

Reply:
Sounds good. I'll look at places where that is called and add an assert or an error to make sure folks do not try to use the filterblast utilities when they are using a remote data base.

Reply:
why should they not be able to do a FilterBlast approach with the web-based version?

Reply:
Ah I thought you were saying it doesn't work yet - I was just thinking of the error as temporary until we know it does work, in v2.